po/snappy.pot

```diff
@@ -367,10 +367,12 @@ msgstr  ""
 msgid   "no interfaces found"
 msgstr  ""
 
+msgid   "No snaps are installed yet. Try 'snap install hello'."
```

Comment:
This should be regenerated with ./update-pot but we can do that easily at release time.